### PR TITLE
[Codewhisperer] add serviceInvocation count field in the %code tracker

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] Local run of `gradlew check` succeeds
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if change is customer facing in the IDE.
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt [hunk 1]
```diff
@@ -601,3 +608,8 @@ data class ResponseContext(
     val sessionId: String,
     val completionType: CodewhispererCompletionType
 )
+
+interface CodeWhispererCodeCompletionServiceListener {
+    fun onSuccess() {}
+    fun onFailure() {}
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:614]: where is onfailure used

**Review by rli — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt (hunk 1 above)
Comment (Will-ShaoHua) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:614]: oh, it hasn't been used, will remove it for now

Comment (Will-ShaoHua) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:614]: and add it back when needed

Comment (Will-ShaoHua) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:614]: sorry Richard, forgot this PR hasn't been reviewed by Andrew

**Review by andrewyuq — APPROVED** [review]

**Pull request merged**